src/main/java/platform/codingnomads/co/springweb/springrestcontrollers/simpledemo/controller/HelloWorldController.java

```diff
@@ -16,6 +19,18 @@ public String hello() {
     public String greeting(@PathVariable(name = "name") String name) {
         return "Hello " + name + "!";
     }
+
+    @RequestMapping(path = "/hello/list", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public String helloList() {
+        List<Integer> integerList = List.of(1, 2, 3, 4);
+        return integerList.toString();
+    }
+
+    @RequestMapping(path = "/hello/pojo", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public String helloTask() {
+        Task task = Task.builder().userId(479).name("Build bars.").description("So many bars.").completed(false).build();
+        return task.toString();
+    }
```

Comment:
This implementation returns:
`Task(id=0, userId=479, name=Build bars., description=So many bars., updatedAt=0, createdAt=0, completed=false)`(string representation of POJO) as response body witch is not `application/json`. 
If it was the goal - all good. 

If the goal was to return actual POJO as `application/json` then you need to change implementation like:
```suggestion
    @RequestMapping(path = "/hello/pojo", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
    public Task helloTask() {
        Task task = Task.builder().userId(479).name("Build bars.").description("So many bars.").completed(false).build();
        return task;
    }
```
Basically, change the return type to be `Task` object rather than `String` object.

Reply:
Yes! That makes a lot more sense. Thanks.